dev/integration_tests/android_embedding_v2_smoke_test/android/build.gradle

```diff
@@ -7,14 +7,14 @@
 // See dev/tools/bin/generate_gradle_lockfiles.dart.
 
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.7.10'
```

Comment:
Is it possible and should we try and use named constants for these version numbers (here and elsewhere)?

Reply:
From testing it looks like constants in the build.gradle get resolved to their values in the lockfiles, so if we did use a constant in the generated `build.gradle`, it would still resolve to a specific value in the lockfile that wouldn't change in line with updates to the constant. So we would still need to re-run the script, which replaces the value in the generated `build.gradle` anyways. 

I do think it could make sense to have the values in the `generate_gradle_lockfiles.dart` templates be constants, especially if we did want them to be the same as the values in templates we use to make a new project. I'll file an issue where we can make the decision.

Reply:
SGTM.